packages/tenant-process/src/model/domain/errors.ts

```diff
@@ -209,7 +209,7 @@ export function tenantIsNotACertifier(
   organizationId: TenantId
 ): ApiError<ErrorCodes> {
   return new ApiError({
-    detail: `Organization ${organizationId} not allowed to assign attributes`,
+    detail: `Tenant ${organizationId} not allowed to assign attributes`,
```

Comment:
We might also align other errors that mention "Organization" and use "Tenant" instead, in this same PR, even though they are not yet involved in API V2 routes - as you prefer